k8s/operators/src/pool/main.rs

```diff
@@ -7,11 +7,18 @@ mod diskpool;
 pub(crate) mod error;
 mod mayastorpool;
 
+use crate::diskpool::client::{
+    create_missing_cr, create_v1beta_cr, discard_older_schema, list_v1alpha1, replace_with_v1beta1,
+    v1beta1_api,
+};
 use chrono::Utc;
 use clap::{Arg, ArgMatches};
 use diskpool::{
-    client::{api, create, ensure_crd},
-    crd::*,
+    client::ensure_crd,
+    new_crd::{
+        CrPoolState as V1beta1PoolState, DiskPool as V1beta1DiskPool,
```

Comment:
use should leave this as unversioned, eg: DiskPool

Reply:
Yes, done. File name is version specific